Makefile

```diff
@@ -0,0 +1,70 @@
+#!make
+
+# Include config variables for custom course
+include envfile
+export $(shell sed 's/=.*//' envfile)
+
+# Execute commands regardless of physical state of files
+.PHONY: build-docs build-dev dev dcourse lcourse  pltest lint unit test docs clean node  
+
+# Install node dependencies (requires node > 10.0)
+node: 
+	npm ci
+
+# Generates a development docker from current directory
+build-dev:
+	docker build -t pltestlocal .
+
+# Show the development version of the prairielearn
+dev: build-dev
+	docker run -it --rm -p 3000:3000 pltestlocal
+
+dcourse: build-dev
+	docker run -it --rm -p 3000:3000 -v ${COURSE_REPO_PATH}:/course prairielearn/prairielearn
+
+# Show a live version of the prairielearn
+live:
+	docker run -it --rm -p 3000:3000 prairielearn/prairielearn
+
+lcourse:
+	docker run -it --rm -p 3000:3000 -v ${COURSE_REPO_PATH}:/course prairielearn/prairielearn
+
+# Remove the docker container
+clean:
+	docker stop pltestlocal; docker rm pltestlocal
```

Comment:
Isn't `pltestlocal` the name of the image, whereas these commands want the name of a container?

Also, maybe we should additionally delete the image with `docker rmi pltestlocal`?